aidants_connect_web/tests/test_forms.py

```diff
@@ -44,7 +44,10 @@ def test_errors_for_blank_items(self):
         def field_test(name_of_field: str, data_set: dict) -> None:
             data_set[name_of_field] = ""
             form = AidantCreationForm(data=data_set)
-            self.assertFalse(form.is_valid())
+            self.assertFalse(
+                form.is_valid(),
+                f"Field {name_of_field} is considered valid, it should not.",
+            )
```

Comment:
On devrait mettre plus souvent des messages d'erreur comme ça :+1:

Reply:
Ça faisait longtemps que je ne l'avais pas fait, et quand j'ai lu `AssertionError: True is not false`, je me suis dit que c'était l'occasion :D